src/ext/java/org/opentripplanner/ext/siri/updater/TransitAlertProvider.java

```diff
@@ -0,0 +1,7 @@
+package org.opentripplanner.ext.siri.updater;
```

Comment:
This interface is required by the core code but lives in the sandbox but is used by GtfsRealtimeAlertsUpdater. Can we move it to the core, too?

Reply:
I moved it to core module now.